src/main/java/frc/robot/constants/fields/HomeField.java

```diff
@@ -1,13 +1,18 @@
 package frc.robot.constants.fields;
 
+import edu.wpi.first.wpilibj.geometry.Translation2d;
 import frc.robot.constants.FieldConstants;
 
 /**
- * Home field dimensions and distances.
+ * Home field way-points. relative to the (0,0) point in PathWeaver. all
```

Comment:
(0, 0) in pathweaver is outside the field.
We can't measure point relative to it...